modin/pandas/__init__.py

```diff
@@ -32,6 +32,7 @@
     NaT,
     PeriodIndex,
     Categorical,
+    options,
```

Comment:
Let's do this in a new PR. Adding options to the `__init__` won't change our internal options. I think it will require a new class.